csharp/ql/test/library-tests/csharp9/PrintAst.expected

```diff
@@ -0,0 +1,139 @@
+#   14|           -1: [TypeMention] int?
+#   14|             1: [TypeMention] int
+#   14|           0: [LocalVariableAccess] access to local variable x1
+#   14|           1: [NullLiteral] null
+#   16|       5: [LocalVariableDeclStmt] ... ...;
+#   16|         0: [LocalVariableDeclAndInitExpr] Nullable<Int32> x2 = ...
+#   16|           -1: [TypeMention] int?
+#   16|             1: [TypeMention] int
+#   16|           0: [LocalVariableAccess] access to local variable x2
+#   16|           1: [ConditionalExpr] ... ? ... : ...
+#   16|             0: [LocalVariableAccess] access to local variable condition
+#   17|             1: [CastExpr] (...) ...
+#   17|               1: [IntLiteral] 12
+#   18|             2: [NullLiteral] null
+#   20|       6: [LocalVariableDeclStmt] ... ...;
+#   20|         0: [LocalVariableDeclAndInitExpr] Nullable<Int32> x3 = ...
+#   20|           -1: [TypeMention] int?
+#   20|             1: [TypeMention] int
+#   20|           0: [LocalVariableAccess] access to local variable x3
+#   20|           1: [ConditionalExpr] ... ? ... : ...
+#   20|             0: [LocalVariableAccess] access to local variable condition
+#   21|             1: [CastExpr] (...) ...
+#   21|               0: [TypeAccess] access to type Nullable<Int32>
+#   21|                 0: [TypeMention] int?
+#   21|                   1: [TypeMention] int
+#   21|               1: [IntLiteral] 12
+#   22|             2: [NullLiteral] null
+#   24|       7: [LocalVariableDeclStmt] ... ...;
+#   24|         0: [LocalVariableDeclAndInitExpr] Nullable<Int32> x4 = ...
+#   24|           -1: [TypeMention] int?
+#   24|             1: [TypeMention] int
+#   24|           0: [LocalVariableAccess] access to local variable x4
+#   24|           1: [ConditionalExpr] ... ? ... : ...
+#   24|             0: [LocalVariableAccess] access to local variable condition
+#   25|             1: [CastExpr] (...) ...
+#   25|               1: [IntLiteral] 12
+#   26|             2: [CastExpr] (...) ...
+#   26|               0: [TypeAccess] access to type Nullable<Int32>
+#   26|                 0: [TypeMention] int?
+#   26|                   1: [TypeMention] int
+#   26|               1: [NullLiteral] null
+#   28|       8: [LocalVariableDeclStmt] ... ...;
+#   28|         0: [LocalVariableDeclAndInitExpr] IEnumerable<Int32> xs0 = ...
+#   28|           -1: [TypeMention] IEnumerable<Int32>
+#   28|             1: [TypeMention] int
+#   28|           0: [LocalVariableAccess] access to local variable xs0
+#   28|           1: [ObjectCreation] object creation of type List<Int32>
+#   28|             -2: [TypeMention] List<Int32>
+#   28|               1: [TypeMention] int
+#   28|             -1: [CollectionInitializer] { ..., ... }
+#   28|               0: [ElementInitializer] call to method Add
+#   28|                 0: [IntLiteral] 0
+#   28|               1: [ElementInitializer] call to method Add
+#   28|                 0: [IntLiteral] 1
```

Comment:
@hvitved There's no cast in this declaration and assignment, so left and right sides are of different types. So I think it's good that we don't have a cast in line 32 either.

Reply:
Thanks for checking this Tamas.